worker.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var commenter = require('./commenter');
+var redisQueue = require('./redisQueue');
+
+var lastCommentTimestamp = 0;
+
+// Process comment tasks
+// Pause for COMMENT_WAIT milleseconds between comment submissions
+redisQueue.process('comment', function(job, done) {
+    var now = Date.now();
+    var waitRemaining = 0;
+
+    var nextCommentTimestamp = lastCommentTimestamp + parseInt(process.env.COMMENT_WAIT);
```

Comment:
Great catch. I was wondering about that too. Do we need to worry about the radix argument in Node? I noticed [JSHint doesn't warn about it anymore](https://jslinterrors.com/missing-radix-parameter).